scripts/check-broken-links.mjs

```diff
@@ -131,43 +208,55 @@ async function checkBrokenLinks() {
   const jsonReport = {
     metrics: {
       domain: OPTIONS.sitemapUrl,
+      initialPathCount: initialPathCount,
       pagesScanned: pagesChecked.length,
       linksChecked: linksChecked.length,
       brokenLinkCount: brokenLinks.length,
       time: formattedTime,
     },
-    brokenLinks: [],
+    brokenLinksByParent: {},
+    brokenLinksByLink: {},
```

Comment:
Added this here to allow for two views into the data. May not be necessary, since we're now providing a CSV which allows this sort of data view.